<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F1Telemetry</class>
 <widget class="QMainWindow" name="F1Telemetry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PXG F1 Telemetry</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="TrackingWidget" name="trackingWidget">
       <attribute name="title">
        <string>Tracking</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout"/>
      </widget>
      <widget class="CompareLapsWidget" name="compareLapsWidget">
       <attribute name="title">
        <string>Analyse Laps</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3"/>
      </widget>
      <widget class="CompareStintsWidget" name="compareStintsWidget">
       <attribute name="title">
        <string>Analyse Stints</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>574</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TrackingWidget</class>
   <extends>QWidget</extends>
   <header>TrackingWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CompareLapsWidget</class>
   <extends>QWidget</extends>
   <header>CompareLapsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CompareStintsWidget</class>
   <extends>QWidget</extends>
   <header>CompareStintsWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
